batch/avro-file/src/main/java/SinkSample.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.hazelcast.jet.IMapJet;
+import com.hazelcast.jet.Jet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.avro.AvroSinks;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sources;
+import model.User;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+
+/**
+ * Demonstrates dumping a map's values to an avro file
```

Comment:
We should use capitalized "Avro" and a period at the end. Also in the SourceSample.